runtime/js/mlBytes.js

```diff
@@ -526,6 +526,12 @@ function caml_bytes_of_array(a) {
   return new MlBytes(4, a, a.length);
 }
```

Comment:
I think we should mark caml_bytes_of_array as deprecated because its semantic is confusing (using / copying its argument)

Reply:
It's still used in the runtime